crates/stc_ts_file_analyzer/src/analyzer/scope/mod.rs

```diff
@@ -2247,6 +2252,16 @@ impl Expander<'_, '_, '_> {
 
                             Type::Mapped(m) => {}
 
+                            Type::Namespace(..) => {
+                                dbg!(&t);
```

Comment:
Please remove this

Reply:
!! my mistake
I'll fix it right away
thank you